config/confighttp/confighttp.go

```diff
@@ -199,6 +199,9 @@ type HTTPServerSettings struct {
 
 	// Auth for this receiver
 	Auth *configauth.Authentication `mapstructure:"auth,omitempty"`
+
+	// Include propagates the client metadata from the incoming requests to the downstream consumers
+	IncludeMetadata bool `mapstructure:"include_metadata,omitempty"`
```

Comment:
Comments in golang should start with the name of the variable.

Reply:
Fixed!